### DISCUSSION TITLE
TalkNet - Polish model zero

### DISCUSSION THREAD

**Discussion by Patryk6M** [NVIDIA-NeMo/NeMo#3180] [Q&A] [answered]
I'm trying to train a polish zero model for TalkNet based on a 30-hour "The Doll".

I am using Colab Pro and this notebook: https://colab.research.google.com/drive/1VqSWRU1H3KIU6au_ojOGFtU0HQPUFa6t

I trained over a week of time to 1500 epochs (about 200 epochs per day) and the model still twists words very much when synthesizing.

Now I read on your site that maybe I am doing something wrong and wasted a week of time with meaningless training:
_Please note that **we don't currently support other languages**, so you have to write your own G2P or extend our G2P in NeMo to support your language. You should also switch to IPA, as we are currently using CMUdict and ARPAbet._

I found a Polish G2P on the internet: https://github.com/MontrealCorpusTools/mfa-models/blob/main/g2p/polish_g2p.zip

I downloaded NeMo-main.zip from https://github.com/NVIDIA/NeMo
but I don't know which files and how to change them to train correctly in Polish.

**Comment by davidmartinrius** [comment]
I am really interested in this. I hope that some developer of the project responds.

**Comment by XuesongYang** [comment, marked as answer]
Hi @davidmartinrius @Patryk6M, we have deprecated TalkNet for a while. I would suggest trying FastPitch + Hifigan for your task.
